src/jit/lsra.cpp

```diff
@@ -5557,6 +5557,7 @@ void LinearScan::allocateRegisters()
                 if (refType == RefTypeUpperVectorSaveDef)
```

Comment:
@CarolEidt  do you mean that we can delete this path under `#if FEATURE_PARTIAL_SIMD_CALLEE_SAVE` and it will fix the issue without any additional changes? I will check.

Reply:
No - I'm saying there are other options, but I'm mostly just explaining why your original comment is rather misleading in characterizing the issues.

Reply:
Thank you, rewritten.